### DISCUSSION TITLE
Self deploy with 401 error

### DISCUSSION THREAD

**Discussion by rafaelspereira1** [anuraghazra/github-readme-stats#3502] [Q&A] [answered]
### Describe the bug

Self deployed in Vercel is giving 401 error when using in personal readme in github.

### Expected behavior

Should be able to see the stats without errors in personal readme

### Screenshots / Live demo link

![readme](https://github.com/anuraghazra/github-readme-stats/assets/90010200/c3f9d600-1b20-446d-bf49-40cb4fe45eab)
![readmerror](https://github.com/anuraghazra/github-readme-stats/assets/90010200/b2cd9197-0f8b-471a-aba8-a728adbdf457)

**Comment by qwerty541** [comment]
Hey, @rafaelspereira1! The data you provided is not enough to draw any conclusions about the causes of the error. Can you please check build/runtime logs of your project on Vercel dashboard? If there will be something show it to me.

**Comment by rafaelspereira1** [comment]
![deployment](https://github.com/anuraghazra/github-readme-stats/assets/90010200/6545c12b-74e6-4894-84ca-c9704acca2e3)
![runtime](https://github.com/anuraghazra/github-readme-stats/assets/90010200/21ef6d2d-9cb2-4b50-b835-73b3b91961fc)

@qwerty541, no errors in build and 401 in runtime

**Reply by qwerty541** [reply, marked as answer]
Try to disable all kinds of deployment protection in project settings.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/cb76468b-eeeb-4c81-ba51-0399736419a3)

**Reply by rafaelspereira1** [reply]
Works now, thanks!  😁

**Reply by qwerty541** [reply]
You're welcome
